manuscript/chapter6-find-time.md

```diff
@@ -61,7 +63,18 @@ Another option is clearly assign tasks to someone else or delegating. This can b
 At the same time help you to avoid burnout, understand what is the amount of tasks that require you and others that can be done from others.  
 As contributor or leader is important to empower others to create better relationships but just do a better team job for the purpose of the project itself.  
 The benefits of the delegating can be very surprising because with empowering others there will be probably new opportunities to grow or change something that you never think of.  
-At the same time this avoid the gatekeeping that sometimes in OSS is a problem that block the project itself also for long time and creating a difficult feeling around the work that you are doing.
+At the same time this avoid the gatekeeping that sometimes in OSS is a problem that block the project itself also for long time and creating a difficult feeling around the work that you are doing.  
+
+## Burnout
+
+About this topics a lot of people with better knowledge then me wrote a lot of books.  
+My experience is basically to understand what are the priorities and define time slot, where after that you don't care except very important things (it is a skills understand what are them).  
+
+Usually burnout has different symptoms: isolation, feeling undervalued, recognition, values conflict, lose motivation and lack of autonomy.  
+There are somethings that you can do quick easy: automatize tasks (I use the rule of 3), understand what others are doing and how, communicate better.
+
+I># Rules of 3
+I> I don't remember where i found it but I have my own version: if it is a task that I do with the computer like 3 times a day or for week always in the same way you need to automatize it. Especially if this task is based on a lot of steps that can creates confusions. In case it is something where you cannot use a computer there are other ways to automatize things in this days with Zapier or IFTTT or if you are a developer you can find some ways.
```

Comment:
*automate

Reply:
I know the „rule of three” in this variant:

The first time you do a task, do it manually.
The second time, refactor it to make it easier to adjust.
The third time, automate the task.

Reply:
for my is kind of already defined that you need to optimize it